### PR TITLE
Feature/image converter

### PR DESCRIPTION
**Pull request by michelle-gh**
Se añadió una clase abstracta converter
Se expandió las funcionalidades del image converter: resize con tipos predefinidos y filtros predefinidos
Se añadio una función para recibir y guardar el archivo entrante en los endpoints.

### CODE REVIEW

File: api/endpoints.py
Comment (RhemJos) [inline api/endpoints.py:1]: The added features and modifications made are not considering the changes made to file downloads.

File: converters/image_to_image/image_converter.py [hunk 1]
```diff
@@ -1,42 +1,53 @@
...
-        return img
-
-    def image_convert(self, resize=None, rotate=None, grayscale=None):
-        img = self.image_resize()
-
-        if rotate is not None:
-            img = self.image_rotate(img, rotate)
+        self.img = Image.open(self.path)
+
+    def resize(self, measures):
```
Comment (RhemJos) [inline converters/image_to_image/image_converter.py]: I think it's better to get the parameters with the names width and height, as they were.

**Review by RhemJos — APPROVED** [review]

**Review by RhemJos — COMMENTED** [review]
The modifications made for endppoints.py are not considering the changes made to file downloads.

**Review by RhemJos — COMMENTED** [review]
The modifications made in endpoints.py are not considering the changes made to file downloads.

**Review by RhemJos — COMMENTED** [review]
Have the modifications been tested to get files for download?

**Review by diegovillarroelsandy — APPROVED** [review]

**Pull request merged**